having <s:select> use CSS form-select by default (since Bootstrap 5)

Hi,

Since Boostrap 5.x, its documentation indicates that there is a custom CSS for HTML elements `<select>`, namely `form-select`. Would it be possible to have it instead of the generic `form-control` when using `<s:select>` and avoid replacing the Struts select.tpl or (ab)using `cssClass='form-select'` each time?

Thanks

Hi,
Just you to now but there is a Maven error due to its pom.xml:
```
        <maven-compiler-plugin.version>3.13.0</maven-compiler-plugin.version>
```
and the plugin configuration is:
```
                <plugin>
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-compiler-plugin</artifactId>
                    <version>${maven-compiler-plugin.version}</version>
                    <configuration>
                        <source>1.7</source>
                        <target>1.7</target>
                    </configuration>
                </plugin>
```
This just gives the following:
```
[INFO] --- maven-compiler-plugin:3.13.0:compile (default-compile) @ struts2-bootstrap-plugin ---
[INFO] Recompiling the module because of changed source code.
[INFO] Compiling 7 source files with javac [debug target 1.7] to target/classes
[INFO] -------------------------------------------------------------
[WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] bootstrap class path not set in conjunction with -source 7
[INFO] 1 warning
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] Source option 7 is no longer supported. Use 8 or later.
[ERROR] Target option 7 is no longer supported. Use 8 or later.
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Struts2 Bootstrap Plugin Parent Module 5.0.5-SNAPSHOT:
[INFO] 
[INFO] Struts2 Bootstrap Plugin Parent Module ............. SUCCESS [  0.796 s]
[INFO] Struts2 Bootstrap Plugin ........................... FAILURE [  0.554 s]
[INFO] Struts2 Bootstrap Showcase ......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.449 s
[INFO] Finished at: 2024-09-05T11:46:45+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:compile (default-compile) on project struts2-bootstrap-plugin: Compilation failure: Compilation failure: 
[ERROR] Source option 7 is no longer supported. Use 8 or later.
[ERROR] Target option 7 is no longer supported. Use 8 or later.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :struts2-bootstrap-plugin
```

--- a/struts2-bootstrap-plugin/src/main/resources/template/bootstrap/select.ftl
+++ b/struts2-bootstrap-plugin/src/main/resources/template/bootstrap/select.ftl
@@ -19,5 +19,6 @@
  */
 -->
 <#include "/${parameters.templateDir}/${parameters.expandTheme}/controlheader.ftl" />
+<#assign formControlClass = "form-select"/>
 <#include "/${parameters.templateDir}/simple/select.ftl" />
 <#include "/${parameters.templateDir}/${parameters.expandTheme}/controlfooter.ftl" />

